src/sagemaker/jumpstart/utils.py

```diff
@@ -1012,24 +1012,37 @@ def get_jumpstart_configs(
     )
 
 
-def get_jumpstart_user_agent_extra_suffix(model_id: str, model_version: str) -> str:
+def get_jumpstart_user_agent_extra_suffix(
+    model_id: Optional[str], model_version: Optional[str], is_hub_content: Optional[bool]
+) -> str:
     """Returns the model-specific user agent string to be added to requests."""
     sagemaker_python_sdk_headers = get_user_agent_extra_suffix()
     jumpstart_specific_suffix = f"md/js_model_id#{model_id} md/js_model_ver#{model_version}"
-    return (
-        sagemaker_python_sdk_headers
-        if os.getenv(constants.ENV_VARIABLE_DISABLE_JUMPSTART_TELEMETRY, None)
-        else f"{sagemaker_python_sdk_headers} {jumpstart_specific_suffix}"
-    )
+    hub_specific_suffix = f"md/js_is_hub_content#{is_hub_content}"
+
+    if os.getenv(constants.ENV_VARIABLE_DISABLE_JUMPSTART_TELEMETRY, None):
+        headers = sagemaker_python_sdk_headers
+    elif model_id is None and model_version is None:
```

Comment:
can we only add the tag `md/js_is_hub_content` if it is a hub content? we don't want to add unnecessary characters to the user agent, there's a char limit

Reply:
thanks changed it in the new revision